### PR TITLE
[PF-2296] Propagate policy created/lastUpdated time through WSM

### PR DESCRIPTION
**Pull request by ddietterich**
This is copying data from TPS objects out through the WSM interface.

### CODE REVIEW

File: openapi/src/parts/tps.yaml [hunk 1]
```diff
@@ -68,6 +68,10 @@ components:
           uniqueItems: true
           items:
             $ref: '#/components/schemas/WsmPolicyObjectId'
+        createdDate:
+          type: string
```
Comment (mikenev) [inline openapi/src/parts/tps.yaml:72]: Do we want to give these the `format: date-time` like we did in WSM?

**Review by mikenev — APPROVED** [review]
Minor comment otherwise lgtm.

File: openapi/src/parts/tps.yaml (hunk 1 above)
Comment (ddietterich) [reply openapi/src/parts/tps.yaml:72]: No, I don't want to do that. The data is already in string form coming out of TPS, so it seems useless to parse and convert it just to convert it back again.

Comment (yuhuyoyo) [reply openapi/src/parts/tps.yaml:72]: what does the client-side output looks like? my only concern is that ui need to convert it?

**Review by yuhuyoyo — APPROVED** [review]

File: openapi/src/parts/tps.yaml (hunk 1 above)
Comment (ddietterich) [reply openapi/src/parts/tps.yaml:72]: It is the standard ISO 8601 format. I'll add that to the documentation.

**Pull request merged**